drivers/sensor/lis2dh/lis2dh.h

```diff
@@ -47,9 +47,9 @@
 #define LIS2DH_ACCEL_XYZ_MASK		BIT_MASK(3)
 
 #define LIS2DH_LP_EN_BIT_MASK		BIT(3)
-#if defined(CONFIG_LIS2DH_POWER_MODE_LOW)
+#if defined(CONFIG_LIS2DH_OPER_MODE_LOW_POWER)
 	#define LIS2DH_LP_EN_BIT	BIT(3)
-#elif defined(CONFIG_LIS2DH_POWER_MODE_NORMAL)
+#else
 	#define LIS2DH_LP_EN_BIT	0
 #endif
```

Comment:
I don't know this driver very well, but I was expecting to see a different gain value for the different resolutions.
Can you explain to me how it is working when changing the resolution among the 3 possibilities?
Have you tested all 3 cases?

Reply:
I have tested all 3 cases. In low power mode we get values with 0x100 step, in normal mode in 0x40 step, in high resolution mode in 0x10 step. We don't have to use different gains, because the result is always "two’s complement left justified".

Reply:
I see.